<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serveSystem</class>
 <widget class="QMainWindow" name="serveSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>收银员</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>结账列表</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>添加商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>100</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>删除商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>180</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>修改商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>260</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>340</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>新建营业额</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>621</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>打卡</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>上机打卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>下机打卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>459</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>收银合计</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>合计 :123.4元</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/png/button_1.png);}
QPushButton:hover{border-image: url(:/png/button_2.png);}
QPushButton:pressed{border-image: url(:/png/button_3.png);}</string>
     </property>
     <property name="text">
      <string>结账</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="text">
           <string>实收 :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>找零 :123.4元</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
